index.js

```diff
@@ -1,9 +1,28 @@
-var _ = require('lodash');
+var index = {}
 
-var data = require('./codes-postaux.json');
+// Index by postalCode value
+require('./communes2016.json').forEach(function (commune) {
+  commune.codesPostaux.forEach(function (codePostal) {
+    if (!(codePostal in index)) {
+      index[codePostal] = [];
+    }
+    index[codePostal].push({
+      nomCommune: commune.nom,
+      codeInsee: commune.code,
+      population: commune.population || 0
+    });
```

Comment:
What about adding the `codePostal` itself? We can let the client store the requested key, but it's often nicer to move around only one single, fully-packaged object.